apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -156,28 +157,20 @@ export class AdministratorResourcesActionBlock extends Component {
 }
 
 export class SpecialAnnouncementActionBlock extends Component {
-  static propTypes = {
-    imageUrl: PropTypes.string.isRequired,
-    heading: PropTypes.string.isRequired,
-    description: PropTypes.string.isRequired,
-    buttons: PropTypes.arrayOf(PropTypes.shape({
-      url: PropTypes.string.isRequired,
-      text: PropTypes.string.isRequired
-    }))
-  };
 
   render() {
-    const { imageUrl, heading, description, buttons } = this.props;
-
     return (
-      <div style={styles.fullWidthNonResponsive}>
```

Comment:
This styling was the only thing really differentiating `SpecialAnnouncementActionBlock` from `TwoColumnActionBlock` and we want the special announcement to be responsive so I removed it. Then, I  customized `SpecialAnnouncementActionBlock` in the pattern of the other variations.